antarest/study/storage/variantstudy/variant_study_service.py

```diff
@@ -155,6 +156,17 @@ def get_commands(
         study = self._get_variant_study(study_id, params)
         return [command.to_dto() for command in study.commands]
 
+    def _check_commands_validity(
+        self, study_id: str, commands: List[CommandDTO]
+    ) -> None:
+        for i, command in enumerate(commands):
+            try:
+                self.command_factory.to_icommand(command)
+            except Exception:
```

Comment:
as a rule, when catching an exception, you should log the error, with exception detail:
```
except Exception as e:
  logger.error('Some message indicating what failed', exc_info=e)
```
it will be needed for debug purpose. also some information may want to be passed to the following rethrow later (in order for the user to understand better what went wrong)